giotto/mapper/cover.py

```diff
@@ -11,8 +14,10 @@
 from .utils._cover import (_check_has_one_column,
                            _remove_empty_and_duplicate_intervals,
                            _validate_kind)
+from ..utils._docs import adapt_fit_transform_docs
 
 
+@adapt_fit_transform_docs
 class OneDimensionalCover(BaseEstimator, TransformerMixin):
     """Covers of one-dimensional data coming from open overlapping intervals.
```

Comment:
Should it be "Cover" instead of "Covers" since we only get one cover?

Reply:
Well spotted!

Reply:
Done.